theme/highlight.css

```diff
@@ -1,9 +0,0 @@
-/*
- * Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
- * For licensing, see LICENSE.md.
- */
-
-.ck-highlight-button svg path:not(.ck-icon__fill) {
-	/* Do not inherit color from parent. */
-	fill: initial;
-}
```

Comment:
You have deleted all file, it shouldn't be empty like in [ckeditor5-widget](https://github.com/ckeditor/ckeditor5-widget/blob/master/theme/widget.css)?

Reply:
AFAIR it has no counterpart in Lark.